### PR TITLE
New package: Samsung.VoiceRecorder version 1.0.1.38

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111983&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/s/Samsung/VoiceRecorder/1.0.1.38/Samsung.VoiceRecorder.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.VoiceRecorder
+PackageVersion: 1.0.1.38
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=1830321&CDCttType=SW&ModelType=C&ModelName=NP-Q1U/P01/SEA&VPath=SW/200801/20080122135939796_Samsung_Voice_Recorder_1.0.1.38.zip
```
Comment (Trenly) [inline manifests/s/Samsung/VoiceRecorder/1.0.1.38/Samsung.VoiceRecorder.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/200801/20080122135939796_Samsung_Voice_Recorder_1.0.1.38.zip
```

File: manifests/s/Samsung/VoiceRecorder/1.0.1.38/Samsung.VoiceRecorder.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: SetUp.exe
+  InstallerSha256: DA4BAA2B64E0C4DDD20A991AFC908331D90395D1F551982785EEF06D59B0A839
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /passive
```
Comment (Trenly) [inline manifests/s/Samsung/VoiceRecorder/1.0.1.38/Samsung.VoiceRecorder.installer.yaml:16]:
```suggestion
  NestedInstallerType: inno
  NestedInstallerFiles:
  - RelativeFilePath: SetUp.exe
  InstallerSha256: DA4BAA2B64E0C4DDD20A991AFC908331D90395D1F551982785EEF06D59B0A839
```

File: manifests/s/Samsung/VoiceRecorder/1.0.1.38/Samsung.VoiceRecorder.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.VoiceRecorder
+PackageVersion: 1.0.1.38
+PackageLocale: en-US
+Publisher: Samsung
+PackageName: Samsung Voice Recorder
```
Comment (Trenly) [inline manifests/s/Samsung/VoiceRecorder/1.0.1.38/Samsung.VoiceRecorder.locale.en-US.yaml:8]:
```suggestion
PackageName: Voice Recorder
```

File: manifests/s/Samsung/VoiceRecorder/1.0.1.38/Samsung.VoiceRecorder.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.VoiceRecorder
+PackageVersion: 1.0.1.38
+PackageLocale: en-US
+Publisher: Samsung
```
Comment (Trenly) [inline manifests/s/Samsung/VoiceRecorder/1.0.1.38/Samsung.VoiceRecorder.locale.en-US.yaml:7]:
```suggestion
Publisher: Samsung Electronics
```

**Review by Trenly — CHANGES_REQUESTED** [review]
image

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**